ingestion/src/metadata/profiler/orm/functions/sum.py

```diff
@@ -32,6 +32,13 @@ def _(element, compiler, **kw):
     return f"SUM(CAST({proc} AS BIGINT))"
 
 
+@compiles(SumFn, Dialects.Trino)
+def _(element, compiler, **kw):
+    """Cast to BIGINT to address cannot cast nan to bigint"""
+    proc = compiler.process(element.clauses, **kw)
+    return f"SUM(TRY_CAST({proc} AS BIGINT))"
```

Comment:
is `TRY_CAST` to handle columns with NaN ?

Reply:
yes